### DISCUSSION TITLE
addy.io GitLab

### DISCUSSION THREAD

**Discussion by E0697** [ImranR98/Obtainium#789] [Q&A] [answered]
When adding https://gitlab.com/Stjin/anonaddy-android/, it defaults to no version detection.

I have added an access token but that didn't help.

Is it a limitation with that specific repo?

**Comment by ImranR98** [comment, marked as answer]
Version detection only supports common version formats. See these comments for details: https://github.com/ImranR98/Obtainium/issues/404#issuecomment-1492007484

**Reply by E0697** [reply]
Very useful comment. Thanks!
